src/bci_build/package/golang/README.md.j2

```diff
@@ -58,26 +75,44 @@ To run the application tests inside a container, use the following command:
 $ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} go test -v
 ```
 
-**Note:** The Golang image should be used as a build environment. For runtime, self-contained Go binaries should use a `scratch` image and for applications that require external dependencies use the `bci-base` image.
+**Note:** The Go image should be used as a build environment. That means
```

Comment:
This note is so big that I feel it should be an entire section somewhere else. Maybe call this section "Deployment" or "Usage in Production".

Reply:
I can split it into "scatch" and "using sle bci" options.